src/tile/index.html

```diff
@@ -0,0 +1,12 @@
+<div class="gb-product-tile">
```

Comment:
Wouldn't it be preferential to programmatically add the class based off of the tag name to the tag itself, thus you would have `<gb-product-tile class="gb-product-tile">` rather than adding an additional layer of structure (which based on previous conversations I had believed you wanted to avoid)